src/storage/logs.js

```diff
@@ -18,13 +18,13 @@ function debug(location) {
 }
 
 function error(location) {
-  return (error, notif) => {
+  return (errorMessage, notif) => {
     const logger = debugConsole(location);
-    logger(error);
+    logger(errorMessage);
     const log = {
       id: uuidv4(),
       timestamp: Date.now(),
-      error: error,
+      error: errorMessage,
       location: location
     };
     db.insert(ERRORS, log);
```

Comment:
the UI won't let me add this note on the specific line, but a few below this there's another reference to `error` that needs to be updated to `errorMessage` as well:

```diff
- notif: notif || error,
+ notif: notif || errorMessage,
```